graphistry/Plottable.py

```diff
@@ -22,17 +26,28 @@
         from sklearn.pipeline import Pipeline
     except:
         Pipeline = Any
+    try:
+        from cugraph import Graph
+        from cugraph import MultiGraph
+        from cugraph import BiPartiteGraph
+    except:
+        Graph = Any
+        MultiGraph = Any
+        BiPartiteGraph = Any
 else:
     UMAP = Any
     Pipeline = Any
+    Graph = Any
+    MultiGraph = Any
+    BiPartiteGraph = Any
 
 
 RenderModesConcrete = Literal["g", "url", "ipython", "databricks", "browser"]
 RENDER_MODE_CONCRETE_VALUES: Set[RenderModesConcrete] = set(["g", "url", "ipython", "databricks", "browser"])
 RenderModes = Union[Literal["auto"], RenderModesConcrete]
 RENDER_MODE_VALUES: Set[RenderModes] = set(["auto", "g", "url", "ipython", "databricks", "browser"])
 
-class Plottable(object):
+class Plottable(Protocol):
```

Comment:
Maybe we take this time to change to a new name `Graph`, and leave `Plottable` as a deprecated alias?

Reply:
oh, I didn't know about `runtime_checkable` sounds good